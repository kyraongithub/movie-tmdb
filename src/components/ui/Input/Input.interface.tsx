export interface InputPropsInterface
  extends React.InputHTMLAttributes<HTMLInputElement> {}
