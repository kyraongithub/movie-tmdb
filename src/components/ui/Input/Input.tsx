import type { InputPropsInterface } from './Input.interface';

const Input = (props: InputPropsInterface) => {
  return <input {...props} />;
};

export default Input;
