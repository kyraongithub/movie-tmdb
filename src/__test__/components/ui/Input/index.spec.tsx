import { render } from '@testing-library/react';
import Input from '../../../../components/ui/Input';

describe('Input', () => {
  it('render Input component', () => {
    const { container } = render(<Input type="text" />);
    expect(container).toMatchSnapshot();
  });
});
