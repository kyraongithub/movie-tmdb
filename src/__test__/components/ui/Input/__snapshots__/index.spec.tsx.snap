// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Input > render Input component 1`] = `
<div>
  <input
    type="text"
  />
</div>
`;
